src/index.js

```diff
@@ -38,16 +38,15 @@ if (!isProduction) {
     app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
+// if (isProduction) {
+//     //remove mongoose conection string, use postgres db fr production
+// } else {
+//     //use postgres db for development
+// }
```

Comment:
These comments don't serve any purpose and are therefore unnecessary, kindly remove them.

Reply:
i'm on it
thanks